Iban move

tperale opened the pull request.

Commit:
Removed IBAN from footer
+ added current year date.

Commit:
Added iban to contact section.

loispostula commented on welcoming/templates/home.html:
```diff
@@ -44,12 +44,12 @@
                 <span class="glyphicon glyphicon-copyright-mark"></span>
-                2015 - Cerkinfo ASBL - BE02 0014 0193 5340 -
+                <script>document.write(new Date().getFullYear())</script>
+                - Cerkinfo ASBL -
```
Je le laisserais quand même en dessous tu sais

tperale replied:
Dans la section contact et en dessous ?

loispostula replied:
ca fait pas de mal je pense

titouanc committed:
[fix] Do not install Django >1.9

With Django 1.10, the following error occurs when migrating:
AttributeError: 'Settings' object has no attribute 'TEMPLATE_CONTEXT_PROCESSORS'

titouanc committed:
[fix] local settings import

titouanc committed:
[fix] Missing migrations

Commit:
Readd the IBAN to the footer.
